js/index.html

```diff
@@ -72,6 +72,9 @@
         </div>
     </div>
 
+    <p style="text-align:center" class="patient" id="patientNameId"> placeholder name
+    </p>
```

Comment:
We need to get Id data, then maybe a landing Menu showing 3-5 patients so we can drill down to the pulse page.
This is just to show the non-pulse data simply.